spec/directions/bake_toc_spec.rb

```diff
@@ -19,51 +20,86 @@
             <span data-type="" itemprop="" class="os-text">Preface</span>
           </h1>
         </div>
-        <div data-type="chapter">
```

Comment:
Can you leave the unit-less `book1` and its spec in and then add another spec that shows that units also work?  That way we can test both; as is the test no longer tests the case without units.

Reply:
oop and this fixed the code coverage issue I was having 😆